Header header

#Pitch roll and yaw in degrees
float64 roll
float64 pitch
float64 yaw

#raw 10-it ADC readings
int16 a0
int16 a1
int16 a2
int16 a3
int16 a6
int16 a7

#Current settings of the motors [-1.0,1.0]
float64 leftMotorSetting
float64 rightMotorSetting